pillar.example

```diff
@@ -19,3 +19,15 @@ samba_sections:
     guest ok: no
     writeable: no
     printable: yes
+  sharename:
+    comment: "This is a shared directory"
+    browseable: yes
+    writeable: yes
+    valid users: @sharegroup
+    force group: sharegroup
+    create mask: 0660
+    directory mask: 2770
+
+samba_users:
+   user1: user1sambapassword
```

Comment:
more properties? uuh, good question, I honestly just need two users on my play-server.

Changing  the value into a map seems like a good idea.

```yaml
 samba_users:
  user1:
    password: blah
    other_option: blubb
```